src/app/selects/basic-ng-select.html

```diff
@@ -0,0 +1,11 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<clr-select>
+    <clr-select-menu>
+        Select Menu
+    </clr-select-menu>
```

Comment:
Now that I see it, how about `<clr-options>` (with an "s") instead of `<clr-select-menu>`? It feels clearer, and "menu" is really off for a select.

Reply:
Yes. I like that. Will update it to `clr-options`. Thanks!